modules/explorer/explorer.go

```diff
@@ -18,27 +17,41 @@ var (
 // Basic structure to store the blockchain. Metadata may also be
 // stored here in the future
 type Explorer struct {
+	// Transaction type counts.
+	transactionCount          uint64
+	siacoinInputCount         uint64
+	siacoinOutputCount        uint64
+	fileContractCount         uint64
+	fileContractRevisionCount uint64
+	storageProofCount         uint64
+	siafundInputCount         uint64
+	siafundOutputCount        uint64
+	minerFeeCount             uint64
+	arbitraryDataCount        uint64
+	transactionSignatureCount uint64
```

Comment:
less certain in this case, but you might be able to embed an `ExplorerStatistics` struct here, like the host does with `HostSettings`